content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,132 @@
+---
+Title: 'html_entity_decode()'
+Description: 'Converts HTML entities to their corresponding characters.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`html_entity_decode()`** function is the opposite of **[htmlentities()](../htmlentities/htmlentities.md)** function in that it converts HTML entities in the string to their corresponding characters.
+
+## Syntax
+
+```pseudo
+htmlentities(string,flags,encoding)
+```
+
+The `html_entity_decode()` function has one required parameter and three optional parameters:
```

Comment:
@caupolicandiaz  I have some solutions like below and wonder what the most suitable solution for this case is.

+Should I just provide information about the string parameter, a little bit about $flag, $encoding, $double_encode then lead users into find more information from php.net website?

+Should I just specify 3-4 constant name with description for the $flag parameter, and 3-4 Charset with Aliases and description for the $encoding parameter? I just feel like the information is too long and I want to lead users to outside resources like w3school, or php.net for more information

+Should I keep all the details like I just wrote and just write the description in my way so that users can understand them easier?